contracts/BondingEvent.sol

```diff
@@ -293,7 +293,7 @@ contract BondingEvent is AccessControl {
     function transferExcessToWallet(uint256 _addedAmount, uint256 _desiredAmount) private {
         uint256 excess = _desiredAmount - _addedAmount;
         if (excess > 0 && excessCollateralWallet != address(0)) {
-            TransferHelper.safeTransfer(OTHER_ADDRESS, excessCollateralWallet, excess);
+            TransferHelper.safeTransfer(SEURO_ADDRESS, excessCollateralWallet, excess);
```

Comment:
that is a good question. i was thinking that there wouldn't be any because of the accuracy differences, but now i'm thinking about it again, it probably is possible. i will add the same conditional transfer for the other token (but it should be must less likely to be used!)

Reply:
ok, I saw your initial comment about transferring seuro instead of usdt. just thinking: why not do both should it come to it.